### PR TITLE
Portfolio-mobile

### PR DESCRIPTION
**Pull request by benjaMatteo96**
Updates:
- Add a semantic HTML.
- Add a CSS file with relatives values.
- Add Linters.
- Add an update to README

### CODE REVIEW

**Comment by benjaMatteo96** [comment]
I really try for several hours to fix that linter error.
- I used npx stylelint "**/*.{css,scss}" --fix
- I search for errors in the CSS file syntaxis
- I delete the my old CSS file and create another one

**Comment by benjaMatteo96** [comment]
I really try for several hours to fix that linter error.
- I used npx stylelint "**/*.{css,scss}" --fix
- I search for errors in the css file sintaxis
- I delete the my old CSS file and create anoter one

**Review by franclobo — COMMENTED** [review]
Hi @benjaMatteo96, 

terminator

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors ❌ 

I'm not sure if the style.css is an empty file because when I try to open I have this 👇🏽 

image

image


## 1. It could be helpful to copy this line in your terminal 👉🏽  `npx stylelint "**/*.{css,scss}" --fix`
## 2. It is probable that the style.css file is corrupted or empty in that case delete it.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by benjaMatteo96** [comment]
Hello @franclobo I installed lfs in my git repo, I think that tool give me errors, but I already uninstalled that. And for the record, I already delete my old CSS file and create another one but the same problem persists.

**Comment by benjaMatteo96** [comment]
@franclobo I just delete the "lfs" folder, and that work :D

**Comment by franclobo** [comment]
> 

Hi, @benjaMatteo96 
I haven't had this issue before and don't know what is the feature that you've installed, but I recommend creating a new branch from the main branch.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        </div>
+    
+        <div id="lets-connect">
+            <h2> Let's Connect</h2>
+            <img src="img/twitter.svg" alt="">
+            <img src="img/Vector (1).png" alt="">
+            <img src="img/Vector.png" alt="">
+            <img src="img/medium.svg" alt="">
+            <img src="img/github.svg" alt="">
+            <img src="img/angellist.svg" alt="">
```
Comment (aamirkhan2478) [inline index.html:39]: - [ ] As I mentioned above :point_up_2: Please stick to the Figma design your icons are very small and also some spacing problems as well so kindly fix it :+1: check below :point_down: the comparison between your design and Figma design :slightly_smiling_face: 

**Your Design**
Screenshot from 2023-04-28 23-47-27

**Figma design**
Screenshot from 2023-04-28 23-47-54

- [ ] Please icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). so kindly use `ul` and `li` for grouping your `icons` :+1:  check below :point_down:  my suggestions :slightly_smiling_face: 

```suggestion
           <ul>
          <li>
            <a href="#">
              <img src="img/twitter.svg" alt="" />
            </a>
          </li>
          <li>
            <a href="#">
              <img src="img/Vector (1).png" alt="" />
            </a>
          </li>
          <li>
            <a href="#">
              <img src="img/Vector.png" alt="" />
            </a>
          </li>
          <li>
            <a href="#">
              <img src="img/medium.svg" alt="" />
            </a>
          </li>
          <li>
            <a href="#">
              <img src="img/github.svg" alt="" />
            </a>
          </li>
          <li>
            <a href="#">
              <img src="img/angellist.svg" alt="" />
            </a>
          </li>
        </ul>
```

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @benjaMatteo96 , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linter passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+#union-logo:hover {
+  border-radius: 2em;
+  background-color: #f5f5f5;
+}
+
+.headline {
+  background-image: url(img/header-shapes\ mobile.png);
+  background-size: cover;
+  overflow: hidden;
+}
```
Comment (IngDavidTM) [inline main.css:27]: - [ ] As per the previous reviewer's request, it is recommended that you utilize Flexbox to properly align the items within the headline section of your website. Flexbox is a powerful CSS layout system that enables you to create flexible and responsive layouts with ease. By implementing Flexbox, you will be able to achieve proper alignment of your headline section items, resulting in a more polished and professional-looking website :+1:.

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @benjaMatteo96 🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**